src/Game/zEntCruiseBubble.cpp

```diff
@@ -452,9 +459,9 @@ void cruise_bubble::start_damaging()
 
 // func_80057684
 #ifndef NONMATCHING
-#pragma GLOBAL_ASM("asm/Game/zEntCruiseBubble.s", "damage_entity__Q213cruise_bubble30_esc__2_unnamed_esc__2_zEntCruiseBubble_cpp_esc__2_FR4xEntRC5xVec3RC5xVec3RC5xVec3fb")
+#pragma GLOBAL_ASM("asm/Game/zEntCruiseBubble.s", "damage_entity__13cruise_bubbleFR4xEntRC5xVec3RC5xVec3RC5xVec3fb")
 #else
-void cruise_bubble::damage_entity(xEnt& ent, const xVec3& loc, const xVec3& dir, const xVec3& hit_norm, float32 radius, uint8 explosive)
+void cruise_bubble::damage_entity(xEnt& ent, const xVec3& loc, const xVec3& dir, const xVec3& hit_norm, float32 radius, bool explosive)
```

Comment:
I think this is supposed to be an unnamed namespace within the cruise_bubble namespace.

```c++
namespace cruise_bubble
{
    namespace
    {
        void damage_entity(xEnt& ent, const xVec3& loc, const xVec3& dir, const xVec3& hit_norm, float32 radius, bool explosive)
        // ...
```

Try that and see if the symbol matches.